lldb/test/API/CMakeLists.txt

```diff
@@ -58,8 +58,13 @@ set(LLDB_TEST_COMMON_ARGS
   -S nm
   -u CXXFLAGS
   -u CFLAGS
+  )
+if (LLDB_ENABLE_SWIFT_SUPPORT)
```

Comment:
Why is this needed in the first place? If the variable doesn't exist it will just be empty. I'm actually in the process of removing this variable altogether.

Reply:
Makes sense. The variable is actually always set (I still need to put all the swift test setup code behind an `if (swift)`) but passing in the swift test parameters doesn't seem to break anything and it will resolve itself once I get around to update the test CMake stuff.